CMakeLists.txt

```diff
@@ -396,11 +396,11 @@ endif ()
 # ----------------------------------------------------------------------------
 # installation directories
 if (OS_WINDOWS)
-  set (RUNTIME_INSTALL_DIR Bin)
-  set (LIBRARY_INSTALL_DIR Lib)
-  set (INCLUDE_INSTALL_DIR Include)
-  set (CONFIG_INSTALL_DIR  CMake)
-  set (PKGCONFIG_INSTALL_DIR)
+  set (RUNTIME_INSTALL_DIR "bin")
+  set (LIBRARY_INSTALL_DIR "lib")
+  set (INCLUDE_INSTALL_DIR "include")
+  set (CONFIG_INSTALL_DIR  "share/${PACKAGE_NAME}")
```

Comment:
`vcpkg` require `share/${PACKAGE_NAME}`, but I can use an easy workaround this and use `lib/cmake/${PACKAGE_NAME}`

CMake should support both options:
```
<prefix>/(lib/<arch>|lib|share)/cmake/<name>*/          (U)
<prefix>/(lib/<arch>|lib|share)/<name>*/                (U)
<prefix>/(lib/<arch>|lib|share)/<name>*/(cmake|CMake)/  (U)
```
https://cmake.org/cmake/help/v3.4/command/find_package.html